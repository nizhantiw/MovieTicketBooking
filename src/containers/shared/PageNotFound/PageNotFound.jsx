import React, { Component } from 'react'

export default class PageNotFound extends Component {
    render() {
        return (
            <div style={{ backgroundColor: "lightcyan"}}>
               <h1> PageNotFound!!</h1>
            </div>
        )
    }
}
