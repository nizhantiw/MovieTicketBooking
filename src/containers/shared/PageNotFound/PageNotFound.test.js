import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer'
import PageNotFound from './PageNotFound'
configure({adapter: new Adapter()});

configure({adapter: new Adapter()});
it("renders correctly", () => {
  const wrapper = shallow(
    <PageNotFound />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders a snapshot', () => {
  const tree = renderer.create(<PageNotFound/>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('should show the text', () => {
    const toggleInstance = shallow(<PageNotFound />);
    const element = toggleInstance.find('div h1');
    expect(element.text()).toMatchSnapshot();
});