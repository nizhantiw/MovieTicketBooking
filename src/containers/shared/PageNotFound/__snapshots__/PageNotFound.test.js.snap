// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "lightcyan",
    }
  }
>
  <h1>
     PageNotFound!!
  </h1>
</div>
`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;

exports[`should show the text 1`] = `" PageNotFound!!"`;
